package.json

```diff
@@ -25,7 +25,6 @@
     "node": ">=14"
```

Comment:
I believe this should be also updated:

https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines

Reply:
Good idea!  Done.